src/api/spec/features/webui/projects_spec.rb

```diff
@@ -0,0 +1,37 @@
+require "browser_helper"
+
+RSpec.feature "Projects", :type => :feature, :js => true do
+  let!(:user) { create(:confirmed_user) }
```

Comment:
So should we stop using Faker and default to more predictable user-, package-, ... names?